docs/src/howto-guides/014_thin_edge_logs.md

```diff
@@ -72,3 +72,37 @@ Collectd logs all the messages at `/var/log/syslog`.
 So, the collectd specific logs can be accessed using the `journalctl` as below
 
 `journalctl -u collectd.service`
+
+## Configuring log levels in `thin-edge.io`
+
+The log levels can be configured for `thin-edge.io` services using either by command line or setting the required log
+level in `system.toml`
+
+### Setting the log level through Cli
+
+The log level can be enabled for a `thin-edge.io` service as below
+
+For example for tedge_mapper:
+
+```shell
+
+sudo -u tedge -- tedge_mapper  --debug c8y
+
+```
+> Note: In a similar way it can be set for all the `thin-edge.io` services.
+  Only `debug` level can be set through cli.
```

Comment:
Technically the `--debug` option enable the `TRACE` log level, so maybe we should mention that.

Reply:
Added that point.